pkgs/servers/holochain-go/default.nix

```diff
@@ -0,0 +1,31 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.2.1
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "holochain-go${version}";
+  version = "v0.1.0-alpha";
+  rev = "a17510b910a7a377441c152b8dccdbae1999f63f";
+
+  goPackagePath = "github.com/holochain/holochain-proto";
+
+
+  src = fetchgit {
+    inherit rev;
+    url = "https://github.com/holochain/holochain-proto";
+    sha256 = "19l29jnr63ximmyn4i4llv2mdwh306c2mpzmx2anj9z12wjpach0";
```

Comment:
Can you use `fetchFromGitHub` since you are not using git submodules?

Reply:
Yup, done